app-directory/i18n/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**"]
```

Comment:
```suggestion
      "outputs": [".next/**", "!.next/cache/**"],
```

Reply:
We need to merge this PR https://github.com/vercel/turbo/pull/4159 cc @tknickman